app/requirements/local.txt

```diff
@@ -2,6 +2,7 @@
 
 psycopg2-binary==2.9.3
 
-black==19.3b
```

Comment:
Changing black version now will make a huge black diff

Reply:
Ok, as we said in #215. Fine with me, if you merge this PR after the workshop :)